h2o-core/src/main/java/water/jdbc/SQLManager.java

```diff
@@ -42,7 +42,7 @@ public class SQLManager {
 
   private static final String DISALLOWED_JDBC_PARAMETERS_PARAM = H2O.OptArgs.SYSTEM_PROP_PREFIX + "sql.jdbc.disallowed.parameters";
 
-  private static final Pattern JDBC_PARAMETERS_REGEX_PATTERN = Pattern.compile("(?i)[?;&]([a-z]+)=");
+  private static final Pattern JDBC_PARAMETERS_REGEX_PATTERN = Pattern.compile("(?i)([a-zA-Z0-9_]+)\\s*=\\s*");
```

Comment:
Wouldn't `"(?i)([a-z]+)\\s*=` suffice? If I understand it correctly you care mostly about the disallowed parameters which do not contain `_` and you're already doing a case-insensitive match (`(?i)` IIRC).